### PR TITLE
update install docs

### PR DESCRIPTION
**Pull request by matyasf**
update documentation on installation and release process

### CODE REVIEW

File: docs/getting-started/releases.md [hunk 1]
```diff
@@ -4,97 +4,17 @@ category: Getting Started
...
-1. Push the merge commit for review: `git push origin HEAD:refs/for/master`
-1. Review the merge commit, test it and merge it.
+Every commit that has functional changes to the `master` branch constitutes as a relase. Ther release process is the following:
+- When the code you want to add is ready on your branch you should open a pull request. 
+- Wait for the pull request to be reviewed and watch any Github notifications about issues/comments by the maintainers.
+- Run `yarn bump` to increase the version automatically based on the commits.
+    - If the code to be merged contains a breaking change, it will result in a major version increase.
+    - If the change is new functionality, it will result in a minor version increase.
+    - Bugfixes, performance changes and pure refactorings will result in a patch release.
+- `git push` the new release script
```
Comment (HerrTopi) [inline docs/getting-started/releases.md:18]:
```suggestion
- `git push` the new release commit
```

File: docs/getting-started/releases.md [hunk 2]
```diff
@@ -4,97 +4,17 @@ category: Getting Started
...
+- Review the changes
```
Comment (HerrTopi) [inline docs/getting-started/releases.md:19]:
```suggestion
- Wait for the review of the release commit
```

File: docs/getting-started/releases.md [hunk 3]
```diff
@@ -4,97 +4,17 @@ category: Getting Started
...
+- Rebase the code into the `master` branch. Jenkins will detect that there is a new version and will update the documentation and publish the new `npm` package.
```
Comment (HerrTopi) [inline docs/getting-started/releases.md:20]:
```suggestion
- (Maintainers only) - Rebase the code into the `master` branch. Jenkins will detect that there is a new version and will update the documentation and publish the new `npm` package.
```

Comment (matyasf) [reply docs/getting-started/releases.md:20]: I've put a note at the top that this whole page is for maintainers only, external contributors cannot make a release

File: docs/getting-started/releases.md [hunk 4]
```diff
@@ -4,97 +4,17 @@ category: Getting Started
...
-1. Verify that the release was published to Npm by running `yarn info [package]@[version]`.
-
-E. __Merge the release into the master branch:__
-
-1. Run `git fetch --all --tags`.
-1. Run `git checkout -B your_local_branch --track origin/master`.
-1. Run `git merge origin/stable`
-1. Push the merge commit for review: `git push origin HEAD:refs/for/master`
-1. Review the merge commit, test it and merge it.
+Every commit that has functional changes to the `master` branch constitutes as a relase. Ther release process is the following:
```
Comment (ke1k0) [inline docs/getting-started/releases.md:11]:
```suggestion
Every commit that has functional changes to the `master` branch constitutes as a release. The release process is the following:
```

File: docs/getting-started/releases.md [hunk 5]
```diff
@@ -4,97 +4,17 @@ category: Getting Started
 order: 8
 ---
 
-## Releases
```
Comment (ke1k0) [inline docs/getting-started/releases.md:7]: This deleted block seems to contain some important info, should we delete them all? What do you think @HerrTopi, do we need any of this?

Comment (matyasf) [reply docs/getting-started/releases.md:7]: As I know most of the functionality was deleted. Also its a bit too step-by-step for me, when someone wants to contribute we can assume that they know Git commands. But I'm open to any suggestion :)

Comment (ke1k0) [reply docs/getting-started/releases.md:7]: I'd say we should keep the info that is still valid. Even the step-by-step info can be useful when a junior joins the team :)

Comment (matyasf) [reply docs/getting-started/releases.md:7]: hmm okay, I bought this argument, I'll rewrite it :)

Comment (matyasf) [reply docs/getting-started/releases.md:7]: I've added it back

**Review by ke1k0 — APPROVED** [review]

**Pull request merged**